src/Game/debugInput.h

```diff
@@ -0,0 +1,16 @@
+
+#pragma once
+
+#include <heap/seadDisposer.h>
+
+class DebugInput {
+    SEAD_SINGLETON_DISPOSER(DebugInput);
+
+public:
+    DebugInput() = default;
+    virtual ~DebugInput() = default;
+
+private:
+    bool t1 = false;
+    u32 t2 = 0;
```

Comment:
```suggestion
    u32 _c = 0;
```
(Note that this is not at offset 8+1 = 9 because of alignment reasons: uint32s are usually 4-byte aligned. The compiler automatically inserts invisible padding bytes to make fields correctly aligned.)